packages/flutter/test/material/expansion_panel_test.dart

```diff
@@ -121,6 +162,46 @@ void main() {
     expect(box.size.height - oldHeight, greaterThanOrEqualTo(100.0)); // 100 + some margin
   });
 
+  testWidgets('ExpansionPanelList does not merge header when canTapOnHeader is false', (WidgetTester tester) async {
+    await tester.pumpWidget(
+      MaterialApp(
+        home: ExpansionPanelListSemanticsTest(),
+      ),
+    );
+    final SemanticsHandle handle = tester.ensureSemantics();
+    await tester.pump();
+    // Make sure it is clickable.
+    await tester.tap(find.text('head1'));
```

Comment:
Why? I thought it is not clickable with canTapOnHeader set to false?

Reply:
canTapOnHeader set to true means tap on header also trigger expand and collapse. The idea is when canTapOnHeader is set to false, user can have their own gesture detector and semantics

Reply:
That information should probably be in the comment above, e.g. instead of "Make sure it is clickable" say "Make sure the user-provided gesture handler can receive the click"